src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -808,6 +814,14 @@ private boolean handleJmxNotificationForCassandra22(
                     .withEndTime(DateTime.now())
                     .withId(segmentId)
                     .build());
+
+            gotSuccessNotification.set(Boolean.TRUE);
```

Comment:
autobox?
ie `completeNotified.set(true);`

Reply:
Just a habit I've taken... I'll switch back to `true`.